test/integration/operator/clustermanager_test.go

```diff
@@ -1183,7 +1183,10 @@ var _ = ginkgo.Describe("ClusterManager Default Mode", ginkgo.Ordered, func() {
 
 		ginkgo.It("should have labels on resources created by clustermanager", func() {
 
-			labels := map[string]string{"app": "clustermanager", "createdByClusterManager": "hub", "test-label": "test-value", "test-label2": "test-value2"}
+			labels := map[string]string{"app": "clustermanager", helpers.HubLabelKey: "hub", "test-label": "test-value", "test-label2": "test-value2"}
+			// app and createdByClusterManager are reserved label keys, and will not be changed to the hub resources.
+			expectedLabels := map[string]string{helpers.HubLabelKey: clusterManagerName, "test-label": "test-value", "test-label2": "test-value2"}
```

Comment:
I do not understand why the expectedLabels contains the hubLabelKey, but does not contain the `app: {{ .ClusterManagerName }}-registration-controller`.